### PR TITLE
Stylus

### PR DESCRIPTION
**Pull request by RogerHaase**

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
first commit: looks ok (just does not really belong into scope of "Stylus" PR).

File: src/moin/templates/base.html [hunk 1]
```diff
@@ -53,10 +53,9 @@
 
                 {%- block theme_stylesheets %}
                     <link media="all" rel="stylesheet" href="{{ url_for('static', filename='css/normalize.css') }}" />
+                    <link media="all" rel="stylesheet" href="{{ url_for('static', filename='css/variables.css') }}" />
                     {% if theme_supp.variables_css() %}
                         <link media="all" rel="stylesheet" href="{{ theme_static('css/variables.css') }}" />
-                    {% else %}
-                        <link media="all" rel="stylesheet" href="{{ url_for('static', filename='css/variables.css') }}" />
```
Comment (ThomasWaldmann) [inline src/moin/templates/base.html:59]: guess the idea of the old code was that a theme's variables.css may or may not @import the default variables.css.

**Comment by ThomasWaldmann** [comment]
is there an issue which this PR is fixing or why are we removing stylus?

File: src/moin/templates/base.html (hunk 1 above)
Comment (RogerHaase) [reply src/moin/templates/base.html:59]: My understanding is browsers load CSS files in parallel, but @import files cannot be loaded until CSS parsing begins. Small benefit.

Old code loaded only 1 variables.css assuming that if a theme had a variables.css it was going to override all the common.css colors; maybe implement a dark theme. New code allows a theme to have a variables.css if it wants one.

**Comment by RogerHaase** [comment]
> is there an issue which this PR is fixing or why are we removing stylus?

#1052 

CSS variables seem easier to use than preprocessors in general. Also more powerful should you choose to modify the variables with Javascript. The only downside to use of CSS variables is IE11 market share.

Long ago I thought I would like using CSS preprocessors, but I now feel they are all time wasters and CSS variables will eventually rule.

**Comment by ThomasWaldmann** [comment]
OK, if we can live without that, feel free to remove. IIRC another nice thing in stylus was python-like syntax. Guess we can't have that with css directly. But if there is too much pain with stylus elsewhere, we can drop it.

I didn't review the css changes in here due to their size.

**Pull request merged**